docs/source/bouncing.rst

```diff
@@ -152,10 +152,42 @@ Valid options are:
 * ``check_haproxy``: Whether to check the local haproxy to make sure this task
   has been registered and discovered.
 
+Understanding How ``bounce_margin_factor`` Affects Bouncing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``bounce_margin_factor`` setting in marathon yaml files controls how
+"aggressive" the bounce is in the face of failures. It only applies to the ``crossover`` bounce method.
+
+With the default setting (1.0) the ``crossover`` bounce will only begin to
```

Comment:
Please do not block-justify your RST source code like this - it turns small changes into large changes.
One sentence per line instead, please.